apps/src/aichat/constants.ts

```diff
@@ -1,4 +1,3 @@
-export const CHAT_COMPLETION_URL = '/openai/chat_completion';
```

Comment:
This endpoint is only used in the `chatApi.ts` file that aichat folks say they don't want in their directory anymore.